plugins/builtin/apm/prometheus/plugin/plugin.go

```diff
@@ -145,6 +163,41 @@ func (a *APMPlugin) QueryMultiple(q string, r sdk.TimeRange) ([]sdk.TimestampedM
 	}
 }
 
+func generateTLSConfig(config map[string]string) (*tls.Config, error) {
+	tlsConfig := tls.Config{}
+
+	// Load the CA certificate if present.
+	caCertPath := config[configKeyCACert]
+	fmt.Println(caCertPath)
```

Comment:
Assuming this is a forgotten debugging line?

Reply:
🤦